README.md

```diff
@@ -176,6 +176,10 @@ If there is an entry without a value, there will be an extra icon too, for displ
 
 The `categorical` legend will simply map the values to their colors.
 
+#### Annotations
```

Comment:
It would be nice if you could go a little bit more into detail here. Take a look at this for example: []Q-table footnotes(https://github.com/nzzdev/Q-table#footnotes)